engine/access/rpc/engine.go

```diff
@@ -72,6 +74,8 @@ func New(log zerolog.Logger,
 	grpcServer := grpc.NewServer(
 		grpc.MaxRecvMsgSize(config.MaxMsgSize),
 		grpc.MaxSendMsgSize(config.MaxMsgSize),
+		grpc.StreamInterceptor(grpc_prometheus.StreamServerInterceptor),
+		grpc.UnaryInterceptor(grpc_prometheus.UnaryServerInterceptor),
```

Comment:
Do you think there's much performance overhead from these interceptors? Just thinking it might be best to only pass them in if `rpcMetricsEnabled == true`